tests/test_backend.py

```diff
@@ -3926,6 +3926,15 @@ def func(input_x, low, high):
                 return tf.identity(res, name=_TFOUTPUT)
             self._run_test_case(func, [_OUTPUT], {_INPUT: input_val, _INPUT1: low_val, _INPUT2: high_val})
 
+    @check_opset_min_version(11, "CumSum")
```

Comment:
Why do we need to add a limitation for CumSum op for this test?

Reply:
i copied the function from above and then made edits (the other test don't need this too then?). I removed the limitation for the test set now.

Reply:
It depends on if ONNX op CumSum is required during the conversion of a tf op. If yes, then we need to limit min opset version to 11 because ONNX supports CumSum op since 11. By adding this, the test case will be ignored if the test suite is running with an ONNX opset lower than 11.

Reply:
MatrixBandPart is a tf op. It doesn't have a requirement for ONNX opset version. The ONNX opset version requirement depends on all ONNX ops we use during the conversion of a TF op.